frappe/geo/country_info.json

```diff
@@ -2913,7 +2913,7 @@
   "currency_fraction_units": 100,
   "isd": "+58"
  },
- "Vietnam": {
+ "Viet Nam": {
```

Comment:
we should probably revert this one, other changes look ok (although not strictly necessary).

Reply:
more: https://politics.stackexchange.com/questions/25642/preferred-or-correct-english-spelling-vietnam-or-viet-nam